docs/source/tutorials/how_to_add_optimization_pass.md

```diff
@@ -39,11 +39,11 @@ takes an `AcceleratorSpec` as input and returns `Dict[str, PassConfigParam]`.
 - `type_` : type of the parameter
 
 - `required` : whether the parameter is required
-
-- `is_object` : whether the parameter is an object/function. If so, this parameter accepts the object or a string with the
+- `category`: The param category. It could be the following values:
+    * `object` : whether the parameter is an object/function. If so, this parameter accepts the object or a string with the
     name of the object in the user script. The type must include `str`.
-
-- `is_path` : whether the parameter is a path. If so, this file/folder will be uploaded to the host system.
+    * `path` : whether the parameter is a path. If so, this file/folder will be uploaded to the host system.
+    * `data`: whether the parameter is a data path, which will be used to do data path normalization based on the data root.
```

Comment:
I saw it can be `none`, in which case we need that case?

Reply:
it is the default value